src/main/java/core/basesyntax/service/impl/ReturnAction.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ActionHandler;
+
+public class ReturnAction implements ActionHandler {
+    public void apply(String fruit, int quantity) {
```

Comment:
The method signature for `apply` should include `throws RuntimeException` to indicate that this method might throw an unchecked exception. This is a part of good coding practices to explicitly declare the potential exception even if it's unchecked.